samples/nrf9160/modem_shell/src/link/link.c

```diff
@@ -408,15 +408,23 @@ static int link_default_pdp_context_auth_set(void)
 	return 0;
 }
 
-static int link_enable_rel14_features(void)
```

Comment:
I would have maybe written two different functions, one for enable and one for disable to make code more readable, but that is just a matter of taste.